### ISSUE TITLE
behavior change of urllib on Python2 from 0.14.x to 0.15.0

### ISSUE THREAD

**Issue by megies** [PythonCharmers/python-future#178]
Return type of `urllib.request.urlopen` changed for some documents from `future` version 0.14.x to 0.15.0 on Python 2:

```python
from __future__ import (absolute_import, division, print_function,
                        unicode_literals)
from future.builtins import *  # NOQA
from future import standard_library, __version__

import sys 

with standard_library.hooks():
    import urllib.parse
    import urllib.request


addr = "http://examples.obspy.org/agfainventory.xml"

print(sys.version)
print("future: %s" % __version__)
print(addr)
response = urllib.request.urlopen(addr)
doc = response.read()
print(type(doc))
```

With future 0.14.x:
```
2.7.10 |Anaconda 2.1.0 (64-bit)| (default, Oct 19 2015, 18:04:42) 
[GCC 4.4.7 20120313 (Red Hat 4.4.7-1)]
future: 0.14.3
http://examples.obspy.org/agfainventory.xml
<type 'str'>
```

With future 0.15.x:
```
2.7.10 |Anaconda 2.1.0 (64-bit)| (default, Oct 19 2015, 18:04:42) 
[GCC 4.4.7 20120313 (Red Hat 4.4.7-1)]
future: 0.15.0
http://examples.obspy.org/agfainventory.xml
<class 'future.types.newbytes.newbytes'>
```

Is this an intended change?

See obspy/obspy#1107.

**Comment by megies** [comment]
Ping?

**Comment by megies** [comment]
@edschofield, ping?

**Comment by megies** [comment]
Well.. fixed on our side with obspy/obspy#1107 but I still think this change is pretty drastic and might have been done inadvertently and might hurt other people as well..?

Well, closing this, as it seems to get no attention anyway.
